lib/RichtextRenderer.astro

```diff
@@ -0,0 +1,38 @@
+---
+import { renderRichText } from "./";
+import type { ISbRichtext, SbBlokData, SbRichTextOptions } from "./dist/types";
+
+import StoryblokComponent from "./StoryblokComponent.astro";
+
+export interface Props {
+  richtextObject: ISbRichtext;
+  richtextOptions?: SbRichTextOptions;
+}
+
+const { richtextObject, richtextOptions } = Astro.props;
+
+const isRichtextEmpty =
```

Comment:
@Baheya @manuelschroederdev I think this can live as a util function in `@storyblok/js`, so other SDKs can benefit and we don't duplicate its logic. What do you think?

Reply:
I agree! Let's postpone this release until that's done then. @Baheya how do you feel about making a PR to `@storyblok/js`? 😉

Reply:
@alexjoverm @manuelschroederdev I'll get on it then, consider it done! 😉